code/game/machinery/door_control.dm

```diff
@@ -201,7 +208,17 @@
 				M.close()
 				return
 
-	icon_state = "launcherbtt"
+	if(secure)
+		icon_state = "is_open"

```

Comment:
This is already taken cared of in the proc.
`/obj/machinery/driver_button/secure/attack_hand(mob/user as mob)`